### PR TITLE
feat: cards skeletons

### PR DESCRIPTION
**Pull request by fgomensoro**
image

### CODE REVIEW

File: src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
-import styles from './GameCardImage.module.css';
+
 import notLiked from '../../../../assets/GameCard/like/not-liked.svg';
```
Comment (elios11) [inline src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx:3]: You can use the assets alias

**Review by elios11 — APPROVED** [review]

File: src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx (hunk 1 above)
Comment (fgomensoro) [reply src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx:3]: yes! but thats not my component. Its Vales

Comment (elios11) [reply src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx:3]: Oh, thought you changed that, my bad!

**Comment by elios11** [comment]
By the way, super nice job, it's looking amazing!

**Review by valselviz — APPROVED** [review]
Looks great!

**Pull request merged**